e2e/vm/additional_disk_test.go

```diff
@@ -37,7 +49,11 @@ var testAdditionalDisk = func(o *option.Option) {
 			gomega.Expect(newImagesOutput).Should(gomega.Equal(oldImagesOutput))
 
 			newPsOutput := command.StdoutStr(o, "ps", "--all", "--format", "{{.Names}}")
-			gomega.Expect(newPsOutput).Should(gomega.Equal(oldPsOutput))
```

Comment:
why this change?

Reply:
My bad, I accidentally deleted it when I was testing :(, will put it back.